src/meta/meta_http_service.h

```diff
@@ -6,11 +6,15 @@
 
 #include <algorithm>
 
+#include <dsn/cpp/json_helper.h>
```

Comment:
you can move this line to the corresponding cpp file

Reply:
This update is for further pull request, I will add structure in `.h` file using json serilization.

Reply:
So I think you can add this line in the future pull request.

Reply:
It is still useful for this pull request, I use `NON_MEMBER_JSON_SERIALIZATION` marco to make structure `start_bulk_load_request` json-serializable. As your comment shows, I can add this in `meta_http_service.cpp`, but considering it will finally move into `.h` file, I did it in this pull request.

Reply:
could you paste the definition of `NON_MEMBER_JSON_SERIALIZATION` here?